### PR TITLE
fixing trivia game

### PR DESCRIPTION
**Pull request by michaone007**

### CODE REVIEW

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 1]
```diff
@@ -28,72 +28,73 @@
...
         }
     }
-    public void changeColor(final View view) {
+    public void nextQuestion() {
         Handler handler = new Handler();
         handler.postDelayed(new Runnable() {
             public void run() {
+                if(index==NUMBER_OF_QUESTIONS){
+                    sendScore();
+                }else
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java]: missing { }

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 2]
```diff
@@ -136,4 +137,15 @@ public static void loadQuestionToScreen(TriviaQuestion m) {
         answer3.setText(m.getAnswerC());
         answer4.setText(m.getAnswerD());
     }
+    private void loopViews() {
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java]: what is it? change name.

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 3]
```diff
@@ -28,72 +28,73 @@
...
-        Log.d(MainActivity.TAG, text);
-        if (boom.get(flash).getCorrectAnswer() == text) {
-            count++;
-            flash++;
+        Log.d(MainActivity.TAG, "flash =" + index);
+        Log.d(MainActivity.TAG, "array size = " + QuestionsArray.size());
+        Log.d(MainActivity.TAG, "correct = "+QuestionsArray.get(index).correctAnswer);
+        Log.d(MainActivity.TAG, "my pick="+text);
+        if (QuestionsArray.get(index).getCorrectAnswer().equals(text)) {
+            index++;
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java]: remove

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 4]
```diff
@@ -28,72 +28,73 @@
...
             view.setBackgroundColor(Color.GREEN);
-            changeColor(view);
+            nextQuestion();
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java]: remove

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 5]
```diff
@@ -28,72 +28,73 @@
...
         } else {
-            wrongCount++;
-            flash++;
             view.setBackgroundColor(Color.RED);
-            changeColor(view);
+            loopViews();
+            wrongCount++;
+            index++;
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java]: remove

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 6]
```diff
@@ -28,72 +28,73 @@
...
+            nextQuestion();
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java]: remove

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 7]
```diff
@@ -28,72 +28,73 @@
...
         }
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java:69]: put these lines down here (outside the if-else)

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 8]
```diff
@@ -28,70 +28,80 @@
...
         } else {
-            wrongCount++;
-            flash++;
             view.setBackgroundColor(Color.RED);
-            changeColor(view);
+            changeToGreen();
+            wrongCount++;
         }
+            nextQuestion();
+            index++;
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java]: opposite order?

Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java]: done

**Pull request merged**